### ISSUE TITLE
[MessageBox]: [A11y][SF] The emphasized button text is not being read by the screen reader

### ISSUE THREAD

**Issue by VidhiNPatel** [SAP/ui5-webcomponents-react#7691]
### Describe the bug

I want to add the accessible-name for the footer emphasized button , but there is not any property to add the accessible name on the emphasized button.

### Reproduction steps

1.Go to https://sap.github.io/ui5-webcomponents-react/v2/?path=/docs/modals-popovers-messagebox--docs
2. Open the MessageBox
3. Run the screen reader and observe that the emphasized button text is not being read by the screen reader

### Expected Behaviour

The screen reader should read the emphasized button text

### UI5 Web Components for React Version

2.13

### UI5 Web Components Version

2.13

### Browser

Chrome

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @VidhiNPatel 

currently the `Button` has a bug that the announcement for the text content is missing if the `design` prop is set (see https://github.com/SAP/ui5-webcomponents/pull/12077). Since a fix has already been merged (https://github.com/SAP/ui5-webcomponents/pull/12077) I guess it will be included in the next release of `@ui5/webcomponents`. So, this issue will be fixed as well then.

To adjust the accessible name of the footer buttons, you can use the `actions` prop to render your own buttons.
